src/vaadin-chart-series.html

```diff
@@ -412,6 +436,7 @@
           return config;
         }
 
+        /** @private */
         __showWarn(propertyName, acceptedValues) {
```

Comment:
`__showWarn()` seems to be called from vaadin-chart so should be `@protected`?

Reply:
Looks like chart tried to access the non-existing method. Copied it there.